docs/book_files/csi_block_storage_kc_pdfs.md

```diff
@@ -8,8 +8,8 @@ To view a PDF file, you need Adobe™ Reader. You can download it at no charge f
 
 |IBM block storage CSI driver publication|Description|PDF download link|
 |----------------------------------------|-----------|-----------------|
-|*Release notes*|This publication describes requirements, compatibility, change log, and known issues information for version 1.6.0.<br />First Edition (June 2021) for version 1.6.0.|![PDF icon](PDF_icon.jpg) [Download](../pdf/IBM_block_storage_CSI_driver_1.6.0_RN.pdf)|
-|*User guide*|This publication describes how to prepare for, install, configure, and use IBM block storage CSI driver.<br />Publication number: SC27-9590-10.|![PDF icon](PDF_icon.jpg) [Download](../pdf/IBM_block_storage_CSI_driver_1.6.0_UG.pdf)|
+|*Release notes*|This publication describes requirements, compatibility, change log, and known issues information for version 1.7.0.<br />First Edition (June 2021) for version 1.7.0.|![PDF icon](PDF_icon.jpg) [Download](../pdf/IBM_block_storage_CSI_driver_1.7.0_RN.pdf)|
+|*User guide*|This publication describes how to prepare for, install, configure, and use IBM block storage CSI driver.<br />First Edition (September 2021).|![PDF icon](PDF_icon.jpg) [Download](../pdf/IBM_block_storage_CSI_driver_1.7.0_UG.pdf)|
```

Comment:
are these links always broken on purpose?

Reply:
they work on IBM Docs and are specifically in the book files folder.

Reply:
are you sure? I don't see them in the book_files folder here

Reply:
csi_block_storage_kc_pdfs.md is specifically in the book_files. the PDF is not.

Reply:
so why do we need it here, if its pdfs aren't here?

Reply:
i need it as part of my source files to go up on IBM Docs. this is how we set it up. things that i need for ibm docs but are not part of the actual content are sitting in this folder. i can't place it anywhere else for version release.

Reply:
so could the `book_files` dir name start with a dot, to clarify that it is only for internal use?

Reply:
I don't know if this can be contained within this release (there is a bunch of linking involved). please open a ticket under my 1.7.0 documentation work and I'll try. if not, i'll defer to next release.

Reply:
done: CSI-3402